src/ui/public/doc_table/components/table_row/cell.html

```diff
@@ -10,4 +10,11 @@
 %>
 <td <%= attributes %>>
   <%= formatted %>
+  <span class="table-cell-filter">
+    <% if (filterable) { %>
+      <i>&nbsp;</i>
+      <i ng-click="inlineFilter($event, '+')" class="fa fa-search-plus" data-column="<%- column %>" tooltip="Filter for value" tooltip-append-to-body="1"></i>
+      <i ng-click="inlineFilter($event, '-')" class="fa fa-search-minus" data-column="<%- column %>" tooltip="Filter out value" tooltip-append-to-body="1"></i>
```

Comment:
Can we make a few changes here:

1. Instead of `<i>` elements, can we use `<span>` elements? (They're more semantic)
2. Let's format this markup according to our [HTML styleguide](https://github.com/elastic/kibana/blob/master/style_guides/html_style_guide.md).
3. Let's add additional classes to the icons to tweak their styles.

The resulting markup would look like this:

```html
<i
  ng-click="inlineFilter($event, '+')"
  class="fa fa-search-plus docTableRowFilterIcon"
  data-column="<%- column %>"
  tooltip="Filter for value"
  tooltip-append-to-body="1"
></i>

<i 
  ng-click="inlineFilter($event, '-')" 
  class="fa fa-search-minus docTableRowFilterIcon" 
  ata-column="<%- column %>" 
  tooltip="Filter out value" 
  tooltip-append-to-body="1"
></i>
```